sdcm/ycsb_thread.py

```diff
@@ -197,11 +198,17 @@ def parse_final_output(result):
         output = {k: str(v) for k, v in output.items()}
         return output
 
+    def db_node_to_query(self, loader):
+        """Select DB node in the same region as loader node to query"""
+        if self.params.get("region_aware_loader", False):
```

Comment:
if we stick with the configurations options, default should be here...

Reply:
right

Reply:
*shouldn't